docs/about/release_notes.rst

```diff
@@ -31,6 +27,119 @@ Release Notes
 Highlighted changes
 ===================
 
+Version 9.0
+-----------
+
+New workflow: Evaluate ensemble
+###############################
+
+In this version of ERT we have introduced a new simulation mode called: Evaluate ensemble.
```

Comment:
I propose leaving out some details. Something like this perhaps:

In this version of ERT, we've introduces a new simulation mode, `Evaluate ensemble`, to address the previously confusing dual functionality of the `Ensemble experiment` simulation mode, which was used both for conducting sensitivity analyses and performing manual updates.
Previously, ERT attempted to automatically discern the user's intention, leading to unpredictable behavior.
To solve this, the Evaluate ensemble mode never overwrites parameters and runs only on ensembles that have parameters but lack responses.
Conversely, Ensemble experiment will now always generate new parameters and create new experiments. 
Each new experiment and ensemble must have a unique name, eliminating the need to delete ERT's internal storage.